src/actors/builtin/cron/cron_actor.go

```diff
@@ -1,11 +1,28 @@
 package cron
 
 import (
+	addr "github.com/filecoin-project/go-address"
 	abi "github.com/filecoin-project/specs/actors/abi"
 	builtin "github.com/filecoin-project/specs/actors/builtin"
 	vmr "github.com/filecoin-project/specs/actors/runtime"
 )
 
+type CronActorState interface {
+	Impl() *CronActorState_I
+}
+
+type CronTableEntry interface {
+	ToAddr() addr.Address
+	MethodNum() abi.MethodNum
+	Impl() *CronTableEntry_I
+}
+
+type CronActorCode interface {
+	Entries() []CronTableEntry
+	EpochTick(r vmr.Runtime)
+	Impl() *CronActorCode_I
+}
```

Comment:
Delete this interface - we'll put in an exports mechanism later (with methodids etc).

In cron_actor.go, create an empty `type CronActor struct{}` as the method receiver (just as go-filecoin and lotus both do now).